examples/template.yml

```diff
@@ -60,8 +60,8 @@ Resources:
     Properties:
       FunctionName:
         Fn::Sub: lambda-function-${EnvName}
-      Description: LambdaFunctioni of nodejs12.x.
-      Runtime: nodejs12.x
+      Description: LambdaFunctioni of nodejs16.x.
+      Runtime: nodejs16.x
```

Comment:
`nodejs16.x` was added in https://github.com/aws-cloudformation/cfn-lint/issues/2263#issuecomment-1126133795, unsupported here?